lua/leetcode/command.lua

```diff
@@ -114,12 +114,14 @@ function cmd.q_run()
     local q = _Lc_questions[_Lc_curr_question]
     if not q then return log.warn("No current question found") end
     q.console:run()
+    if require("leetcode.config").toggle_console_on_run then q.console:toggle() end
 end
 
 function cmd.q_submit()
     local q = _Lc_questions[_Lc_curr_question]
     if not q then return log.warn("No current question found") end
     q.console:submit()
+    if require("leetcode.config").toggle_console_on_submit then q.console:toggle() end
```

Comment:
move this to [runner](https://github.com/kawre/leetcode.nvim/blob/master/lua/leetcode/runner/init.lua)
```suggestion
    if require("leetcode.config").console.open_on_runcode then question.console:show() end
```